evap/evaluation/management/commands/send_reminders.py

```diff
@@ -49,16 +49,13 @@ def send_student_reminders():
     @staticmethod
     def send_textanswer_reminders():
         if datetime.date.today().weekday() in settings.TEXTANSWER_REVIEW_REMINDER_WEEKDAYS:
-            evaluations = [
-                evaluation
-                for evaluation in Evaluation.objects.filter(state=Evaluation.State.EVALUATED)
-                if evaluation.textanswer_review_state == Evaluation.TextAnswerReviewState.REVIEW_URGENT
-            ]
-            if not evaluations:
+            evaluation_url_tuples = Evaluation.get_evaluation_url_tuples_to_review_urgent()
+            if not evaluation_url_tuples:
                 logger.info("no evaluations require a reminder about text answer review.")
                 return
-            evaluations = sorted(evaluations, key=lambda evaluation: evaluation.full_name)
+            evaluation_url_tuples = sorted(evaluation_url_tuples, key=lambda evaluation: evaluation[0].full_name)
```

Comment:
Could / Should the sorting be part of `get_evaluation_url_tuples_to_review_urgent`?